src/window/src/lib.rs

```diff
@@ -21,54 +35,93 @@ pub struct Size {
     pub height: u32,
 }
 
+/// Convinience conversion between an array of size two, and a size structure.
```

Comment:
"Convenience"

Btw, I don't think we need doc comments on the impls unless there is something unusual going on there.